library_builders.sh

```diff
@@ -339,11 +371,16 @@ fi
 
 install_buildessentials
 build_zlib
-build_blosc
```

Comment:
Here it's probably the opposite. Since this still builds ADIOS1, should be put Blosc1 back in?

Reply:
Discussed in person:
We always skipped ADIOS1 on mac x-compile. Now we skip it also for Linux with ppc64le.

Forgot to remove Blosc1 on Windows (never had ADIOS1 support).